lib/developer_portal/lib/liquid/template/fallback_resolver_no_prefix.rb

```diff
@@ -3,18 +3,9 @@
 module Liquid
   class Template
     class FallbackResolverNoPrefix < FallbackResolver
-
-      def initialize(path = DeveloperPortal::VIEW_PATH)
-        super
-      end
-
       def _find_all(name, prefix, partial, details, key = nil, locals = [])
-        path = build_path(name, prefix = nil, partial)
-
-        # force just liquid format
-        details = details.merge(handlers: [:liquid])
-
-        query(path, details, details[:formats], locals, cache: !!key)
+        # force just liquid format and set an empty prefix
+        super(name, '', partial, details.merge(handlers: [:liquid]), key, locals)
```

Comment:
Could you explain this? In particular it is not clear why do we force an empty `prefix` while previously the method used the prefix passed.

Reply:
It's not very clear to me either :sweat_smile: 

Basically, it has to do with the structure of the template files... We have some "shared" template files that are loaded from the file system, if they are not present in the provider's CMS, and as they are shared, they are not prefixed with the prefix, corresponding to the controller.

I actually tried removing this `FallbackResolverNoPrefix` class, but had some templates not being loaded.

For example, there is a template `lib/developer_portal/app/views/developer_portal/_submenu.html.liquid`, which is kind of "shared", and can be included as a partial in different templates. And in this case, we need to remove the prefix that the controller adds (e.g. `prefix: "login"` for the LoginController) to get the template resolved.

Reply:
Makes sense. Thanks I couldn't figure out how id it work in the past so that's why I asked. Could it have been related to the default `path` value? Anyway, I'm fine with it.

Reply:
Yeah, I guess they are related in the sense that the path is constructed as: "path" + "prefix" + "name", so all of the values should be correct for the right template to be found.